locales/en-US/notes.properties

```diff
@@ -108,6 +108,10 @@ themeLegend=Theme
 defaultThemeTitle=Default
 darkThemeTitle=Dark
 
+locationLegend=Open Notes in...
```

Comment:
As a general note, always use `…` instead of `...`

Having said that, `…` seems wrong for the context. It's not a button label requiring more info from the user or opening a dialog, it's a label for the two radio options.

Reply:
Thanks @flodolo, I've gone ahead and removed the ellipsis entirely.